### ISSUE TITLE
[Feature Request] Dedicated Server support

### ISSUE THREAD

**Issue by CatMeat** [Kesomannen/gale#173]
The ability to add Dedicated Servers, to allow for easy updating of server-side mods. Primarily for the steamcmd dedicated server installs, that do not use the Steam GUI. Further support could be provided with command line arguments that run GALE in no GUI mode and update all plugins in the selected profile, then exit back to console. This would allow updating of mods to be automated via a batch or script file as part of the dedicated server start/restart process. Happy to discuss this further and will volunteer as tester should this be considered for inclusion.

**Comment by Kesomannen** [comment]
I've seen this feature in r2modman but not sure exactly what it even does differently? Tried searching r2modman's source files but couldn't find anything

**Comment by AbstractMelon** [comment]
I was looking at this and saw 2 things.
 First of: The post looks AI generated and puts stuff like GALE in all caps
 Second: I looked at r2mm and all it does it just add the dedicated server like a normal game with zero changes

**Comment by Kesomannen** [comment]
> I looked at r2mm and all it does it just add the dedicated server like a normal game with zero changes

Yeah, that's what I also saw. Seems like this is less of a "feature" and more a way to use the mod maanger?

**Comment by NullPointerCollection** [comment]
For valheim dedicated server, r2mm launches the executable for the server instead of the normal game. That's just an example.

**Comment by NullPointerCollection** [comment]
To add some more info here, it's helpful to be able to load mods for a dedicated server instance of valheim through a mod manager.  It's probably the only reason I still have r2mm installed on my computer.  I use it for testing mods I make and it's helpful to support people who are looking for assistance with running a dedicated server since it can rule out a lot of issues introduced by human error.
